test/integration/scheduler/framework_test.go

```diff
@@ -1080,12 +1080,11 @@ func TestReservePluginUnreserve(t *testing.T) {
 					}
 				}
 			}
-
-			preBindPlugin.reset()
-			for _, pl := range reservePlugins {
-				pl.reset()
-			}
-			testutils.CleanupPods(testCtx.ClientSet, t, []*v1.Pod{pod})
+			// preBindPlugin.reset()
+			// for _, pl := range reservePlugins {
+			// 	pl.reset()
+			// }
+			// testutils.CleanupPods(testCtx.ClientSet, t, []*v1.Pod{pod})
```

Comment:
This shouldn't be commented out, however.

Reply:
Yes, it should be kept.